apps/demos/menuMeta.json

```diff
@@ -2076,6 +2076,19 @@
             ]
           }
         ]
+      },
+      {
+        "Name": "AIIntagration",
```

Comment:
We definitely need to update the demo name. Waiting for PMs decision

Reply:
[de8a31a1ab3c157e14e58585a2adf7a0cd4a4d82]